lib/viewcomponents/vertex.jsx

```diff
@@ -33,13 +34,22 @@ DIRECTION_TO_LABEL_POSITION[CardinalDirection.WEST] = PortLabelPosition.RIGHT;
 
 var Vertex = React.createClass({
 
+  getInitialState: function() {
+    return {
+      mouseHover: false
```

Comment:
This feels like the wrong place to be doing hovering stuff. 
Can the UnpinButton manage hovering itself? Better yet, can you just do this in CSS?